spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "GET #new" do
```

Comment:
If you aren't going to write tests for the `new` endpoint, then it's probably a good idea to just remove this block for now.